### PR TITLE
Added a readme.md first version

### PR DESCRIPTION
**Pull request by FisSher**
Created the first version of a readme to introduce users on how to use the template

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Curriculum Vitae template
+
+This repository serves as a template project designed to simplify the process of hosting your Curriculum Vitae (CV) on GitHub Pages. With this template, you can effortlessly showcase your professional profile and accomplishments to potential employers, collaborators, or anyone interested in learning more about you.
```
Comment (Umutayb) [inline readme.md]: Potential clients* instead of employees.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+3. In the "Build and deployment" section, choose the source as "GitHub Actions".
+4. Save your changes.
+
+Your CV website will now be automatically built and hosted on GitHub Pages. You can customize the content, layout, and styling of your CV by modifying the files in your repository. Share your professional profile with the world effortlessly using this GitHub Pages template.
+
+To access your CV, you can click in the "deployments" subsection on the right part of the code tab.
+
+## Editing your CV
+
+To edit your CV content:
```
Comment (Umutayb) [inline readme.md:40]: Can we elaborate on this part? Maybe explain each field in .yml.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Navigate to the "\_data" folder in your repository.
+2. Open the data.yml file to edit your template information.
+3. Replace the profile picture with your own image (100x100) located in /assets/images/.
+
+## Credits
+
+Theme: [sharu725](https://github.com/sharu725/online-cv)
+
+Development: [Umutayb](https://github.com/Umutayb)
```
Comment (Umutayb) [inline readme.md]: Add yourself in here!

**Review by Umutayb — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (FisSher) [reply readme.md]: done :D

File: readme.md (hunk 3 above)
Comment (FisSher) [reply readme.md]: This is done, I extended the names or it would make no sense

File: readme.md (hunk 2 above)
Comment (FisSher) [reply readme.md:40]: Added a longer explanation

**Pull request closed without merging**